code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -182,7 +182,7 @@
 
 	return cached_results[id] ? REACTING : NO_REACTION
 
-//fusion: a terrible idea that was fun but broken. Now reworked to be less broken and more interesting.
+//fusion: a terrible idea that was fun but broken. Now reworked to be less broken and more interesting. Again.
```

Comment:
it keeps happening

Reply:
I'm predicting in the far flung future this comment will actually make up the majority of reactions.dm due to continuous fusion reworks.